user_data.sh

```diff
@@ -18,6 +18,10 @@ pip install --upgrade awscli
 ##############
 
 cat <<"EOF" > /home/${ssh_user}/update_ssh_authorized_keys.sh
+cat <<EOF > /home/${ssh_user}/.ssh/config
+Host *
+    StrictHostKeyChecking no
+EOF
```

Comment:
This block should be not inside line 20, but outside of file `update_ssh_authorized_keys.sh`.